website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -200,3 +200,7 @@ fatal: reference is not a tree: e845be54e6dc72342d5a8f814c8b3316ee220312
 ```
 
 Double-check that your PR isn't trying to merge using a commit that belongs to a fork of the repository attached to your dbt project.
+
+### CI Job Not Triggering for Virtual Private dbt Customers ###
+
+Your Git provider needs to have connectivity to your dbt Cloud account in order to trigger jobs on dbt Cloud using the API. If you are an Enterprise customer using Virtual Private dbt and availing to security features like ingress PrivateLink or IP Allowlisting, registering CI hooks may not be available and will likely fail silently. 
```

Comment:
Sugg for clarity and also suggestion below in case the first sentence isn't accurate:

'your Git provider will need to be able to connect to your dbt Cloud account.'  

For the Virtual Private dbt sentence - is this just a limitation on ci jobs for virtual private users? is the below suggestion correct that ci hooks can cause a job to fail silently?

```suggestion
To trigger jobs on dbt Cloud using the [API](/docs/dbt-cloud-apis/admin-cloud-api), your Git provider needs to connect to your dbt Cloud account.  

If you're on a Virtual Private dbt Enterprise plan using security features like ingress PrivateLink or IP Allowlisting, registering CI hooks may not be available and can cause the job to fail silently. 
```